GrpcClient/EmployeeGrpcRepository.cs

```diff
@@ -0,0 +1,286 @@
+﻿using Grpc.Core;
+using Grpc.Net.Client;
+using RepositoryContracts;
+
+namespace GrpcClient;
+
+public class EmployeeGrpcRepository : IEmployeeRepository
+{
+    private string _grpcAddress { get; set; }
+
+    public EmployeeGrpcRepository()
+    {
+        _grpcAddress = "http://192.168.140.143:50051";
+    }
+
+    public async Task<Entities.Employee> AddAsync(Entities.Employee employee)
+    {
+        using var channel = GrpcChannel.ForAddress(_grpcAddress); 
```

Comment:
Shouldn't 'using' be used with stuff at the top of the file? what exactly is being assigned to the channel property?

Reply:
Yes u are right, but it can be also used like this

Reply:
So what exactly is being assigned to the channel property?